src/configuration/service.cpp

```diff
@@ -218,10 +218,6 @@ namespace mtconnect {
             usage(1);
           }
         }
-        else
```

Comment:
If we run it without argument, it'll crash with exception. We may need to add code to handle that situation.